tests/basecli_test.go

```diff
@@ -0,0 +1,222 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	//"strings"
+	"testing"
+	"time"
+)
+
+// Tests assume the `basecoind` and `basecli` binaries
+// have been built and are located in `./build`
+
+// TODO remove test dirs if tests are successful
+
+var (
+	basecoind = "build/basecoind"
+	basecli   = "build/basecli"
+
+	basecoindDir = "./tmp-basecoind-tests"
+	basecliDir   = "./tmp-basecli-tests"
+
+	ACCOUNTS = []string{"alice", "bob", "charlie", "igor"}
+	alice    = ACCOUNTS[0]
+	bob      = ACCOUNTS[1]
+	charlie  = ACCOUNTS[2]
+	igor     = ACCOUNTS[3]
+)
+
+func gopath() string {
+	return filepath.Join(os.Getenv("GOPATH"), "src", "github.com", "cosmos", "cosmos-sdk")
```

Comment:
More like my comment below - this looks like it should be a global var not a function